### ISSUE TITLE
Game crashes a second after startup

### ISSUE THREAD

**Issue by t0phatt** [Drewol/unnamed-sdvx-clone#78]
All i got in the log .txt file was this:
[23:24:18][Info] Starting task "Application Setup"
[23:24:19][Warning] No joysticks found
[23:24:19][Info] Starting task "Audio Init"
[23:24:19][Info] Finished task "Audio Init" in  51 ms
[23:24:19][Info] Starting task "GL Init"
[23:24:19][Error] Failed to create OpenGL context: Could not create GL context: The operation completed successfully.

[23:24:19][Error] Failed to create OpenGL context
[23:24:19][Info] Finished task "GL Init" in  452 ms
[23:24:19][Info] Finished task "Application Setup" in  1353 ms
[23:24:19][Info] Starting task "Application Cleanup"
[23:24:19][Info] Finished task "Application Cleanup" in  34 ms

**Comment by Drewol** [comment]
What operating system and what hardware are you using?

**Comment by t0phatt** [comment]
Windows 10, and if you mean GPU and CPU that'll be some low-spec stuff here:
Intel(R) Core(TM) i3 CPU   M 370 2.40 GHz
Intel(R) HD Graphics (Core i3) with 64 MB of Display Memory/VRAM (according to dxdiag)

**Comment by Drewol** [comment]
yeah, that's quite low end and also kind of old indeed... I'm not sure the main version of the game is going to run on that but I could build a windows version of the embedded branch and send that to you, might have some more luck with that version.

**Comment by t0phatt** [comment]
please do, i really want to try out this SDVX sim atleast for the time being since i can't afford to try the pc version of Gravity Wars and k-shoot is a bit lacking

**Comment by Drewol** [comment]
Sorry for the delay but here's a build that you might have some more luck with https://drewol.s-ul.eu/yLFRtWvj.zip

**Comment by t0phatt** [comment]
I got a different error this time, being:
[20:12:07][Info] Starting task "Application Setup"
[20:12:07][Warning] No joysticks found
[20:12:07][Info] Starting task "Audio Init"
[20:12:07][Info] Finished task "Audio Init" in  45 ms
[20:12:07][Info] Starting task "GL Init"
[20:12:07][Error] Failed to create OpenGL context: Could not initialize OpenGL / GLES library
[20:12:07][Error] Failed to create OpenGL context
[20:12:07][Info] Finished task "GL Init" in  3 ms
[20:12:07][Info] Finished task "Application Setup" in  243 ms
[20:12:07][Info] Starting task "Application Cleanup"
[20:12:07][Info] Average FPS: inf
[20:12:07][Info] Finished task "Application Cleanup" in  65 ms

If you can't help me beyond this point it's fine, and i'd like to thank you for trying. I'll atleast look for some better parts and use a slightly more capable desktop,

**Comment by Drewol** [comment]
The minimum specs are on paper OpenGL 3.3 for the regular version and kind of OpenGL ES 2.0 for the embedded version. I use those vague terms because I've seen hardware that says it doesn't support 3.3 run the game fine and the embedded version uses one function of ES 3.0 but it runs in an ES 2.0 context but the embedded version does run on raspberry pi and that's the main goal of that version.

**Comment by NickEckardt** [comment]
I had this issue too, and I fixed this by installing the proprietary Nvidia drivers for my machine. 

Try running `glxinfo`, that crashed for me with the same error before I installed the drivers.

**Comment by Hinara** [comment]
Not really a crash has USC close properly.
Seems to be related to material drivers rather than USC
